### ISSUE TITLE
Uninstalling Sometimes Limits Weapon Menu to 1 Weapon

### ISSUE THREAD

**Issue by LethalPlacebo** [serpentiem/ddmk#97]
The "gun switcher" option is great! Having DMC5 level customization over your loadout in DMC3 is fantastic, thanks for that! The only issue is that after uninstalling DDMK (wanted to get Style UI back, having the beta installed made it disappear), if you go to the "equip" menu, you can only choose one gun. You can't switch during gameplay either, you're just locked to only having one gun. Only one can be highlighted in the menu and the other slot just stays empty. I have a backup save, so not a big deal for me, but just thought you should know.

**Comment by serpentiem** [comment]
Thanks for reporting. That's a side effect of how the Weapon Switcher works. There's also a repair option in the tools menu to fix this. I'm currently working on a new design.

**Comment by LethalPlacebo** [comment]
Oh, I'm sorry! I wasn't aware there was a repair option. I can confirm that the "repair" option worked, though I had to make sure to click the menu option and use a Divinity statue in-mission, otherwise it would crash when I switched to the other gun. It seems to have reverted E&I to level 1 (I made sure not to click the menu option that does that) but I have the red orbs to cover it anyway. Thanks for this!
